### PR TITLE
updated react-look package (new syntax + features)

### PR DESCRIPTION
**Pull request by rofrischmann**

### CODE REVIEW

**Comment by MicheleBertoli** [comment]
Thank you very much, @rofrischmann

I tried to run the new demo and what I got is:
```html
<button style="background-color:#ff0000;width:320px;padding:20px;border-radius:5px;border:none;outline:none;:hover:[object Object];:active:[object Object];@media (max-width: 480px):[object Object];" data-reactid=".0.0">Click me!</button>
```
(states and media queries not parsed)

Am I doing something wrong? Does it work for you?

File: react-look/button.js [hunk 1]
```diff
@@ -1,41 +1,40 @@
...
-  }
   render() {
     return (
-      <div look="container">
-        <button look="button">Click me!</button>
+      <div look={styles.container}>
+        <button look={styles.button}>Click me!</button>
       </div>
-    );
+      );
```
Comment (MicheleBertoli) [inline react-look/button.js]: Not needed, thanks

**Comment by rofrischmann** [comment]
Hey @MicheleBertoli,
Thanks for mentioning, actually my formatter (esformatter) added these spaces. 
I forgot to change the package as you need to use `react-look/dom` (or configure your own) for DOM-specific mixins (media queries, pseudo classes).

**Comment by MicheleBertoli** [comment]
Awesome, thanks @rofrischmann

**Pull request merged**